### PR TITLE
feat: ui/ux improvements, console error fixes, added Total Value Locked on DEX

### PR DESCRIPTION
**Pull request by thedoublejay**
1. Improvements on page titles, now we have consistent page titles for all pages. (e.g, Create Wallet - DeFi App). The App name is always appended at the end. App name is also translated. Tested on English and German. In case we need to change the format in the future, we just need to change the utility method.
2. Fixed console error issue on EllipsisText Length
3. Fixed console error issue on **inputmode** (should be **inputMode**)
4. Fixed missing translation for UTXOStoAccount 
5. Fixed overlapping issue on Add Receive Address page
6. Fixed console error issue on re-render for Transaction List page
7. Added "Total Value Locked" on DEX page (Same as explorer)

### CODE REVIEW

File: webapp/src/containers/WalletPage/components/ReceivePage/PaymentRequestList/PaymentRequestList.module.scss [hunk 1]
```diff
@@ -30,3 +31,7 @@
 .colorRed {
   color: $red;
 }
+
+.addressContainer {
+  max-width: 175px;
+}
```
Comment (Sumit570) [inline webapp/src/containers/WalletPage/components/ReceivePage/PaymentRequestList/PaymentRequestList.module.scss:37]: Add new line at end here.

Comment (thedoublejay) [reply webapp/src/containers/WalletPage/components/ReceivePage/PaymentRequestList/PaymentRequestList.module.scss]: updated

**Comment by aentan** [comment]
Please avoid classes like `.colorRed`. We are using Bootstrap and if you have done your homework and have some understanding of the front end CSS framework we used, you'd know that Bootstrap already provide color utility classes: https://getbootstrap.com/docs/5.0/utilities/colors/#color

So use `.text-danger` instead. Willy-nilly adding new ad-hoc classes adds a FE tech debt and headaches down the line.

**Comment by thedoublejay** [comment]
> Please avoid classes like `.colorRed`. We are using Bootstrap and if you have done your homework and have some understanding of the front end CSS framework we used, you'd know that Bootstrap already provide color utility classes: https://getbootstrap.com/docs/5.0/utilities/colors/#color
> 
> So use `.text-danger` instead. Willy-nilly adding new ad-hoc classes adds a FE tech debt and headaches down the line.

@aentan I didn't add it. It was already there, I just changed the html element. Will be changing it to `text-danger`.

**Review by monstrobishi — APPROVED** [review]

**Pull request merged**